node/actors/network/src/lib.rs

```diff
@@ -133,9 +128,6 @@ impl Runner {
                 Ok(())
             });
 
-            // Update QC batches in the background.
-            s.spawn(self.net.gossip.run_batch_qc_finder(ctx));
```

Comment:
What looks for quorums now? 👀

Reply:
`Controller::wait_for_qc()` should be used. I've added documentation to `attestation::Controller`.